content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -59,6 +59,24 @@ Our current process is as follows:
 
 - If one or more issues that were planned for an iteration are looking to **not get finished** (which includes testing) in the [current iteration](https://github.com/orgs/sourcegraph/projects/200/views/1) (while maintaining sustainable work practices) the assignee **raises this as soon as possible asynchronously** to the team (including the PM and EM), e.g. on the GitHub issue or Slack. These issues then **become _proposed_ issues for the next iteration** (meaning nothing carries over automatically, but we also don't just drop and forget missed issues).
 
+## Project Leads
+
+Larger issues that involve multiple domains may benefit from a "project lead". A project lead assumes full responsibility for gathering all requirements to complete an issue. The project lead is expected to have full understanding of the issue and pull in any resources or team members needed to complete the task.
```

Comment:
Do we want to call these "issues" or "projects"? (Confusion with GH issues, or is this intentional/there will be per project issues as well?) 

Also, do they necessarily have to involve multiple domains? (Monorepo improvements feels like primarily a backend project, for example?) 

```suggestion
Larger projects, especially ones that involve multiple domains, may benefit from a "project lead". A project lead assumes full responsibility for gathering all requirements to complete an issue. The project lead is expected to have full understanding of the project and pull in any resources or team members needed to complete the task.
```